modules/Kconfig.mcuboot

```diff
@@ -1,11 +1,51 @@
-config SIGN_IMAGES
-	bool "Sign images for MCUBoot"
-	default y
-	depends on MCUBOOT_BUILD_STRATEGY_FROM_SOURCE \
-		   || BOOT_SIGNATURE_KEY_FILE != ""
+config BOOT_USE_MIN_PARTITION_SIZE
```

Comment:
Nothing wrong on this line.

Is this `modules/Kconfi.mcuboot` the best name and location for this file ?

I'm concerned that developers will not notice that this file is specific for the mcuboot sample itself, but might think it's for the mcuboot module.

Would `modules/mcuboot/sample/Kconfig.mcuboot` or `modules/mcuboot/boot/zephyr/Kconfig` (ref: https://github.com/nrfconnect/sdk-mcuboot/tree/main/boot/zephyr) be cleaner options ?

An even better proposal would be welcome.

Reply:
I went with your first proposal. I'm not able to come up with anything better.